### DISCUSSION TITLE
updated to quarto cli 1.6 and lost preview content in website blog listing pages

### DISCUSSION THREAD

**Discussion by chad-bennett** [quarto-dev/quarto-cli#11738] [Q&A]
I'm building a simple blog website using Quarto. Was using old version of Quarto in VSCode on Mac. Installed an update to 1.6 using Homebrew. 

Previously, the blog main listing page would show title, an image, and a snippet of content for each post. Since the update, the snippet of content has disappeared. Not sure how to bring it back. 

The snippet was the first part of the content from each post. I've tried adjusting the **max-description-length** in the YAML but that doesn't help. I've pored over the docs and guides and can't find any other setting to adjust or toggle to flip. 

Including a screenshot of the design in progress showing where the missing content would be. Also posting an image from the Quarto guides with the missing content elements highlighted. 

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/932e2bdb-1552-4b5c-9f82-af2f7c07934c" />

![Image](https://github.com/user-attachments/assets/5c6f6dd7-3b72-4f89-a0fa-6928e26aff6e)

**Comment by chad-bennett** [comment]
OK. Tried something right after posting this and it works. Just add **description: "content"** in YAML and it works. No idea how to use snippet of content like it did before. This is a functional solution.

**Comment by cderv** [comment]
> No idea how to use snippet of content like it did before. This is a functional solution.

Can you share an example we can use of what you did before and worked ? 

Thanks.

**Reply by chad-bennett** [reply]
Here's a random post before the update. There's no description in the YAML. The main blog listing page would include the content in the page, in this case the text in the column-margin. After the update, none of that would show. I'm guessing I was working with a glitch before and didn't know it. The update fixed the glitch. Now I'm including a description in the YAML and it is working fine. 

`
---
title: "week 11 of 2023"
date: "2023-03-19"
categories: [2023]
format:
  html:
    toc: true
    toc-title: page content
    smooth-scroll: true
---

# week 11 of 2023

![cute cyborg cat](robokitty.jpg)\

::: {.column-margin}
[andreu zaragoza](https://www.instagram.com/andreuzaragoza/) has a great illustration style. love the colors and halftone textures.
:::
`

**Comment by mcanouil** [comment]
I’m going to move this as a discussion as it’s likely an issue with a custom listing.
